.gitignore

```diff
@@ -29,8 +29,7 @@ coverage.xml
 target/
 bin
 tmp/
-go.mod
-go.sum
+vendor/
```

Comment:
What's the motivation to ignore `vendor`? I'd prefer git to complain if it exists.

Reply:
what would be the use case there? the vendor folder should always be created from the go mod file. I don't see a reason to remove the folder before committing some changes.